scripts/un/sdg/util.py

```diff
@@ -95,57 +91,80 @@
     'OBS_STATUS', 'RELEASE_STATUS', 'RELEASE_NAME'
 }
 
-# Supported Regions.
-# TODO: Add other regions.
-REGIONS = {
-    1: 'Earth',
-    2: 'africa',
-    5: 'southamerica',
-    9: 'oceania',
-    10: 'antarctica',
-    21: 'northamerica',
-    142: 'asia',
-    150: 'europe',
+# Series where zero should be treated as null and removed.
+ZERO_NULL = {
+    'SE_ACS_CMPTR',
+    'SE_ACS_H2O',
+    'SE_AGP_CPRA',
+    'SE_ALP_CPLR',
+    'SE_AWP_CPRA',
+    'SE_ACC_HNDWSH',
+    'SE_INF_DSBL',
+    'SE_TOT_CPLR',
+    'SE_TRA_GRDL',
+    'SE_ACS_INTNT',
 }
 
+# Footnote text indicated that a zero point should be treated as null and removed.
+ZERO_NULL_TEXT = 'This data point is NIL for the submitting nation.'
 
-def get_country_map(file):
-    ''' Creates map of M49 -> ISO-alpha3 for countries.
-
-  Args:
-    file: Path to input file.
-
-  Returns:
-    Country map.
-  '''
-    with open(file) as f:
-        places = {}
-        reader = csv.DictReader(f, delimiter='\t')
-        for row in reader:
-            if not row['ISO-alpha3 code']:  # Only countries for now.
-                continue
-            places[int(row['M49 code'])] = row['ISO-alpha3 code']
-    return places
-
-
-PLACES = get_country_map(os.path.join(module_dir_, 'm49.csv'))
-
+# Variables that should be dropped.
+DROP_VARIABLE = {'VC_DTH_TOTPT'}
 
-def get_city_map(file):
-    ''' Creates map of name -> dcid for supported cities.
-
-  Args:
-    file: Path to input file.
-
-  Returns:
-    City map.
-  '''
-    with open(file) as f:
-        reader = csv.DictReader(f)
-        return {row['name']: row['dcid'] for row in reader}
+# Series that should be dropped.
```

Comment:
Add context for why?  A TODO to follow up...

Reply:
done